apiserver/test/cluster/redis/redis.yaml

```diff
@@ -46,7 +46,7 @@ spec:
     spec:
       containers:
         - name: redis
-          image: redis:5.0.8
+          image: redis:7.4.0
```

Comment:
nit: next time can you put the Redis version change in a separate PR so it's easier to find and revert?

Reply:
Sure! Thanks for reminding.